src/drivers/cmakeFileApiDriver.ts

```diff
@@ -2,18 +2,18 @@ import * as api from '@cmt/api';
 import { ConfigureTrigger } from '@cmt/cmakeTools';
 import { ExecutableTarget } from '@cmt/api';
 import { CMakeCache } from '@cmt/cache';
-import { CMakeExecutable } from '@cmt/cmake/cmake-executable';
+import { CMakeExecutable } from '@cmt/cmake/cmakeExecutable';
 import { ConfigurationReader } from '@cmt/config';
-import * as index_api from '@cmt/drivers/cmakefileapi/api';
+import * as index_api from '@cmt/drivers/cmakefileapi/cmakeFileApi';
```

Comment:
It looks kind of odd to have cmakeFileApi in the path twice.  Should we remove the `cmakefileapi` folder and merge `api.ts` and `api_helper.ts` into a single `cmakeFileApi.ts` in the driver folder instead?

Reply:
`index_api` should be renamed to `fileApi` or something.